test/types/change_stream.test-d.ts

```diff
@@ -0,0 +1,9 @@
+import { expectError } from 'tsd';
+
+import type { ChangeStreamOptions } from '../../src';
+
+declare const changeStreamOptions: ChangeStreamOptions;
+
+// The change stream spec says that we cannot throw an error for invalid values to `fullDocument`
+// for future compatability.  This means we must leave `fullDocument` as type string.
+expectError<'updateLookup' | undefined>(changeStreamOptions.fullDocument);
```

Comment:
would an `expectType<string>(changeStreamOptions.fullDocument)` be better here? my concern is that the `expectError` would still pass if we restricted `fullDocument` as long as we didn't restrict it to explicitly `'updateLookup' | undefined`, if I understand the tsd assertions correctly

Reply:
Yeah, you're right.  We should instead test for `string` exactly